test/parallel/test-url-parse-invalid-input.js

```diff
@@ -12,9 +12,13 @@ const url = require('url');
   0.0,
   0,
   [],
-  {}
-].forEach(function(val) {
-  assert.throws(function() { url.parse(val); }, TypeError);
+  {},
+  () => {},
+  Symbol('foo')
+].forEach((val) => {
+  assert.throws(() => { url.parse(val); },
+                /^TypeError: Parameter "url" must be a string, not (undefined|boolean|number|object|function|symbol)$/);
 });
 
-assert.throws(function() { url.parse('http://%E0%A4%A@fail'); }, /^URIError: URI malformed$/);
+assert.throws(() => { url.parse('http://%E0%A4%A@fail'); },
```

Comment:
very minor nit: the `{` and `}` around the body of the arrow function here can be omitted but isn't necessary.